docs/cli.md

```diff
@@ -74,7 +74,7 @@ You can adjust defaults by passing arguments to `node-pg-migrate`:
 - `migrations-table` (`t`) - The table storing which migrations have been run (defaults to `pgmigrations`)
 - `ignore-pattern` - Regex pattern for file names to ignore (e.g. `ignore_file|\..*|.*\.spec\.js`)
 - `migration-filename-format` - Choose prefix of file - either UTC (`20200605075829074`) or timestamp (`1591343909074`)
-- `migration-file-language` (`j`) - Language of the migration file to create (`js` or `ts`)
+- `migration-file-language` (`j`) - Language of the migration file to create (`js`, `sql`, or `ts`)
```

Comment:
At least we should order these extensions by prio:
```suggestion
- `migration-file-language` (`j`) - Language of the migration file to create (`js`, `ts`, or `sql`)
```

Reply:
Agreed on this, I just went alpha by default.